packages/adblocker-webextension/src/index.ts

```diff
@@ -128,19 +132,74 @@ export function updateResponseHeadersWithCSP(
   return { responseHeaders };
 }
 
+const READABLE_MIME_TYPES = new Set([
+  'application/javascript',
+  'application/json',
+  'application/mpegurl',
+  'application/vnd.api+json',
+  'application/vnd.apple.mpegurl',
+  'application/vnd.apple.mpegurl.audio',
+  'application/x-javascript',
+  'application/x-mpegurl',
+  'application/xhtml+xml',
+  'application/xml',
+  'audio/mpegurl',
+  'audio/x-mpegurl',
+]);
+export const MAXIMUM_RESPONSE_BUFFER_SIZE = 10 * 1024 * 1024;
+
+export function shouldApplyReplaceSelectors(
+  request: Request,
+  details: OnHeadersReceivedDetailsType,
+): boolean {
+  const contentTypeHeader = details.responseHeaders?.find(
```

Comment:
lets move this lookup below the content-length check

Reply:
Updated in [`bead6e9` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/bead6e965e01aab9d95de53b1f0c5f2efbdc3385)